che-theia-factory/package.json

```diff
@@ -10,14 +10,14 @@
   ],
   "dependencies": {
     "@eclipse-che/workspace-client": "latest",
-    "@theia/core": "0.3.12",
-    "@theia/editor": "0.3.12",
-    "@theia/git": "0.3.12",
+    "@theia/core": "~0.3.13",
+    "@theia/editor": "~0.3.13",
+    "@theia/git": "~0.3.13",
     "axios": "0.18.0"
   },
   "devDependencies": {
-    "rimraf": "2.6.2",
-    "typescript": "2.7.2"
+    "rimraf": "^2.6.2",
+    "typescript": "^2.9.2"
```

Comment:
we can't use `^` as we need CQ for each dependency

Reply:
OK thanks ... but how do we do with all the dependencies that comes with Theia ?

Reply:
@evidolob I think you told me that this is devDependencies and we have v+above CQ for it? Am I right?

Reply:
I can remove the carets  ^. Will it be fine to merge then ? or do we need to wait for CQ approval ?

Reply:
I will ping PMC for TypeScript, should be quick to approve